### ISSUE TITLE
How do you override php-fpm.conf?

### ISSUE THREAD

**Issue by otravers** [wodby/wordpress-php#1]
I need to increase the value for pm.max_children because my PHP container crashes when I'm trying to send emails with th WP/Mailhog combo, and the log shows "WARNING: [pool app] server reached max_children setting (4), consider raising it"

I copied your php-fpm.conf file and tried to mount it from my docker compose file:

```
volumes:
 - ./client/php-init/conf/php-fpm.conf:/etc/php7/conf.d/php-fpm.conf
```

I tried a couple different paths but phpinfo shows this isn't picked up, I'm already overriding some php.ini settings with a custom.ini file, but despite googling this for a good while I don't know how to do the same for the php-fpm settings, and I'm also unsure whether I could but these settings in an .ini file instead of ,conf.

**Comment by pprishchepa** [comment]
The `php-fpm.conf` file is in the `/etc/php7` directory, so you should override it via:

```
volumes:
 - <your file path>:/etc/php7/php-fpm.conf
```

**Comment by otravers** [comment]
Thanks, this resolves this issue. I see now that I could have found that path information in the image's Dockerfile:
https://github.com/Wodby/wordpress-php/blob/master/Dockerfile

**Comment by csandanov** [comment]
See the list of available env vars available in the new 2.0 version https://github.com/wodby/php
